src/go/configmanager/flags/flags.go

```diff
@@ -109,6 +109,7 @@ var (
 	// Flags for testing purpose.
 	SkipJwtAuthnFilter       = flag.Bool("skip_jwt_authn_filter", false, "skip jwt authn filter, for test purpose")
 	SkipServiceControlFilter = flag.Bool("skip_service_control_filter", false, "skip service control filter, for test purpose")
+	SmCallRetryConfigs       = flag.String("service_management_call_retry_configs", "", `Set retry times and retry intervals for different status calling service management by json string. For example, setting retry 30 times with 3sec interval for status 429 is {"429":{"RetryNum":30,"RetryInterval":3000000000}}. This flag is used for test only`)
```

Comment:
Do we need to expose a flag for this? Can't we unit test with just the option? Not a fan of passing a JSON object via a flag.

Reply:
Yeap, I don't like to expose such flag either.   We can just define a global map, that test code can modify it.

Reply:
Done. The integration test won't have negative test.